apps/api/e2e/tests/leaderboard.test.ts

```diff
@@ -366,241 +366,4 @@ describe("Leaderboard API", () => {
     expect(agents[0]?.voteCount).toBe(3);
     expect(agents[1]?.voteCount).toBe(1);
   });
-
-  test("should sort global leaderboard by different fields", async () => {
```

Comment:
why did we remove this test? we need to be able to sort the `/leaderboard` endpoint with query params, and it looks like you removed the `sort` feature altogether? at the very least, sort by rank/score

Reply:
for context: our original leaderboards had all columns sortable. imo the "new" leaderboard designs regressed and removed that sorting feature. we should definitely still allow the existing sorting params.

Reply:
for reference, other sites with leaderboards:
- allows sorting: https://lmarena.ai/leaderboard
- only shows tops: https://openrouter.ai/rankings